test/absorption_comparison_funcs.jl

```diff
@@ -106,7 +107,22 @@ function _semi_realisitic_dens(ne::F, fion::F= 0.02, HydrogenMassFrac::F= 0.76)
     (nH_I, nH, ρ)
 end
 
-# Now actually define the functions that compute the opacities in the form comparable with Gray05
+# Gray05 uses some confusing terminology and notation.
+# - he defines the "atomic absorption coefficient" as the area per absorber and represent it with
+#   the variable α. For reference, Rybicki & Lightman call this same quantity the cross-section and
+#   represent it with the variable σ_ν.
+# - he defines the "continuous absorption coefficient per neutral hydrogen atom" and denotes it
+#   with κ. Rybicki & Lightman don't explicitly define this quantity, but it's compatible with
+#   their nomenclature. R&L would express this quantity as α_ν/nH_I (in their notation)
```

Comment:
This quantity is actually the same as the cross section!

Reply:
Not exactly. For continuum absorption of He and H_2^+, Gray still defines  the defines the "continuous absorption coefficient per neutral hydrogen atom", κ = α_ν/nH_I. 

Plus, the idea of an absorption coefficient for free-free absorption is something of a murky subject. In detail, the relevant physical quantity for something like He I free-free absorption is technically a "cross-section per free-free electron" for He II and has a dimensionality of `length^5`. As we discussed earlier today, people sometimes employ the Saha equation to transform that quantity to effectively be a cross-section per He I particle.